fog/api/src/extra.rs

```diff
@@ -0,0 +1,60 @@
+use core::convert::TryFrom;
```

Comment:
why is this file called `extra.rs`?

Reply:
this is this like, one off function that is getting the key out of the report and isn't validating anything, so in that sense like, clients aren't really supposed to use it.

I don't know, can you think of a better name?

Reply:
Oh I see.  I thought maybe you had mistyped and it was supposed to be `extract.rs` so what about that?

Reply:
so actually, how about `report_parse.rs`? we called the error `ReportParseError` so at least that would match